streams/src/main/java/org/apache/kafka/streams/processor/internals/TaskManager.java

```diff
@@ -138,7 +142,7 @@ void handleCorruption(final Map<TaskId, Set<TopicPartition>> taskWithChangelogs)
             // this call is idempotent so even if the task is only CREATED we can still call it
             changelogReader.remove(task.changelogPartitions());
 
-            // mark corrupted partitions to not be checkpointed, and then close the task as dirty
+            // mark corrupted partitions to not be checkpointed, and then close the task and revive the task
```

Comment:
We call `closeClean` below -- just fixing the comment here for now (\cc @guozhangwang)